docker-compose.yml-example

```diff
@@ -3,7 +3,7 @@ version: "3.5"
 services:
   web:
     image: ${docker_repo:?err}/kopano_web
-    container_name: web
+    container_name: ${FQDN}
```

Comment:
Imho this will only cause issues and makes it impossible to resolve the fqdn externally - strongly suggest to keep this web.